.github/workflows/release.yml

```
@@ -17,3 +17,33 @@ jobs:
           build_prereleases: true
           ssh_key: ${{ secrets.DEPLOY_KEY }}
           composer_dir: src/usr/local/php/unraid-tailscale-utils/
+
+      - name: Actions for Discord (Release)
+        uses: Ilshidur/action-discord@0.4.0
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.TAILSCALE_DISCORD_WEBHOOK }}
+        with:
+          args: |
```

actionlint (1.7.7)

property "version" is not defined in object type {}

(expression)